app/models/post.rb

```diff
@@ -248,7 +254,7 @@ def has_content_warnings?
 
   def reply_count
     return read_attribute(:reply_count) if has_attribute?(:reply_count)
-    replies.count
+    replies.where.not(reply_order: 0).count
```

Comment:
I think this is redundant with the above scope on `replies`

Reply:
For example, here! because here doesn't match the post.replies pattern!